cmd/pipeline/deploy.go

```diff
@@ -227,11 +193,11 @@ func getPipelineName(repository string) string {
 	return model.TranslateURLToName(repository)
 }
 
-func (pc *Command) streamPipelineLogs(ctx context.Context, name, actionName string) error {
-	return pc.okClient.Pipeline().StreamLogs(ctx, name, actionName)
+func (pc *Command) streamPipelineLogs(ctx context.Context, name, namespace, actionName string) error {
```

Comment:
@teresaromero Be aware of this change regarding your refactor of the stream logs logic